syntax = "proto3";
import "common.proto";
package normal;
option cc_generic_services = true;

message TeamTestRequest
{
}

message TeamTestResponse
{
	ClientError error = 1;
}


service PlayerTeamService
{
  rpc EnterScene (TeamTestRequest) returns (TeamTestResponse);
}